### PR TITLE
added Get method

### PR DESCRIPTION
**Pull request by ilgityildirim**

### CODE REVIEW

**Review by tmclane — APPROVED** [review]

**Comment by leaanthony** [comment]
Please make this PR against `develop`. Thanks!

**Comment by tmclane** [comment]
Would you please rebase on top of develop and remove the merge develop branch into the feature branch please?

'''
git reset --hard HEAD~1
git rebase origin/develop
git push origin +branch
'''

**Comment by ilgityildirim** [comment]
@tmclane ready.
Sorry if you got notifications 2-3 times, my IDE went nuts on me, had to switch to CLI.

File: runtime/store.go [hunk 1]
```diff
@@ -291,3 +291,8 @@ func (s *Store) Update(updater interface{}) {
 		s.errorHandler(err)
 	}
 }
+
+// Get returns the value of the data that's kept in the current state / Store
+func (s *Store) Get() interface{} {
+	return s.data
```
Comment (leaanthony) [inline runtime/store.go:297]: This will return a `reflect.Value`. I suspect you wanted to return `s.data.Interface()`?

Comment (ilgityildirim) [reply runtime/store.go:297]: Ah very true.

**Pull request merged**